frontend/app/components/letter-list.tsx

```diff
@@ -0,0 +1,58 @@
+import React, { useEffect, useState } from 'react';
+
+import { Link } from '@remix-run/react';
+
+export interface Letter {
+  id: string;
+  subject: string;
+  dateSent: string;
+  referenceId: string;
+}
+
+export interface LetterListProps {
+  letters: Letter[];
+  i18n: { subject: string; dateSent: string; referenceId: string };
+}
```

Comment:
```suggestion
export interface LetterListProps {
  letters: Letter[];
  subject: string;
  dateSent: string;
  referenceId: string;
}
```

Or, maybe have the component look up these strings itself (insead of passing them as props).